elements/rh-navigation-primary/rh-navigation-primary.css

```diff
@@ -375,4 +380,46 @@
       }
     }
   }
+
+  &.logo-text-hide-at-xs #logo svg {
```

Comment:
While ordinarily we should prefer shadow classes, because of SSR, we should find an alternative in the case

Ideally, this would all be done in CSS

Reply:
@bennypowers I'm not sure I follow can you reword what you are asking?

Reply:
This new attribute,

If it changes the shadow root

RHDC engineers will have to calculate that change in their template

Until they're able to run lit SSR on each page

When we spoke with them about nav1 being simple enough to fake lit SSR by running it on your laptop with the only two, string-forwarding attributes given twig {{expression}} values,..

Reply:
This sorta throws a 🐒 🔧 in that plan

So let's try to do it with CSS only, since anyways it's a presentational thing

Reply:
After more discussion, we have removed the configurable attribute. Now there is only the 1440px breakpoint for hiding the logo text. Below 1440px, the nav will always hide the text of the default logo.